resources/js/Pages/Category/Index.js

```diff
@@ -116,7 +116,11 @@ export default function Index({auth}) {
                                             {categories.map((item) => (
                                                 <tr key={item.id} className='loaded' id={'item-' + item.id}>
                                                     <td className="px-6 py-4 whitespace-nowrap text-sm font-bold text-gray-800">{item.id}</td>
-                                                    <td className="px-6 py-4 whitespace-nowrap text-sm font-bold text-gray-800">{item.name}</td>
+                                                    <td className="whitespace-nowrap">
+                                                    <span className={"badge " + item.color}>
```

Comment:
I understand the logic here, can we store in the DB just the color name without prefix? so that it's not coupled with the UI
for example instead of color = badge-red => color = red
and then in the UI here we can prefix it with `badge badge-{color}`
this is insure if we want to make change to the CSS classes we don't break the existing functionality

Reply:
Updated as you suggested